<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>xkcd</class>
 <widget class="QWidget" name="xkcd">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>XKCD</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWebView" name="webView">
     <property name="url">
      <url>
       <string>about:blank</string>
      </url>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextBrowser" name="altText">
     <property name="minimumSize">
      <size>
       <width>50</width>
       <height>60</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>234234</width>
       <height>234234</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QProgressBar" name="progressBar">
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="alt">
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>150</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Alt Text</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="minimumSize">
        <size>
         <width>110</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>150</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Back to Menu</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="next">
       <property name="minimumSize">
        <size>
         <width>110</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>150</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>&lt;- Previous</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="minimumSize">
        <size>
         <width>110</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>300</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Random Comic</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="previous">
       <property name="minimumSize">
        <size>
         <width>110</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>150</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Next-&gt; </string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebView</class>
   <extends>QWidget</extends>
   <header>QtWebKit/QWebView</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>alt</sender>
   <signal>clicked()</signal>
   <receiver>xkcd</receiver>
   <slot>alt()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>103</x>
     <y>451</y>
    </hint>
    <hint type="destinationlabel">
     <x>156</x>
     <y>475</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>next</sender>
   <signal>clicked()</signal>
   <receiver>xkcd</receiver>
   <slot>prev()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>522</x>
     <y>453</y>
    </hint>
    <hint type="destinationlabel">
     <x>471</x>
     <y>474</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>previous</sender>
   <signal>clicked()</signal>
   <receiver>xkcd</receiver>
   <slot>next()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>690</x>
     <y>455</y>
    </hint>
    <hint type="destinationlabel">
     <x>626</x>
     <y>477</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>xkcd</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>406</x>
     <y>450</y>
    </hint>
    <hint type="destinationlabel">
     <x>284</x>
     <y>473</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>xkcd</receiver>
   <slot>randComic()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>610</x>
     <y>453</y>
    </hint>
    <hint type="destinationlabel">
     <x>534</x>
     <y>475</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>webView</sender>
   <signal>loadFinished(bool)</signal>
   <receiver>xkcd</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>83</x>
     <y>214</y>
    </hint>
    <hint type="destinationlabel">
     <x>4</x>
     <y>300</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>alt()</slot>
  <slot>next()</slot>
  <slot>prev()</slot>
  <slot>showMain()</slot>
  <slot>randComic()</slot>
 </slots>
</ui>
